src/svg/qsvggenerator.cpp

```diff
@@ -78,7 +81,7 @@ static void translate_dashPattern(const QVector<qreal> &pattern, qreal width, QS
 
     // Note that SVG operates in absolute lengths, whereas Qt uses a length/width ratio.
     for (qreal entry : pattern)
-        *pattern_string += QString::fromLatin1("%1,").arg(entry * width);
+		*pattern_string += QString::fromLatin1("%1,").arg(entry);
```

Comment:
You may want to use spaces for indentation instead of tabs in this file if it is already consistently using spaces.